Source/itemdat.h

```diff
@@ -325,19 +325,19 @@ enum unique_base_item : int8_t {
 enum class ItemSpecialEffect {
 	// clang-format off
 	None                   = 0,
-	Infravision            = 1 << 0,
+	Unused0                = 1 << 0,

```

Comment:
why not just remove them, they have values so won't change things.

Reply:
There were already some `Unknown*` values that also weren't used, so I opted to be consistent. I can delete all of them.